src/driver_servo.cpp

```diff
@@ -35,7 +35,7 @@ const bool kServoDriverDebugEnabled = false;
 
 bool ServoDriver::ProcessConfig(const DriverConfig& config) {
   ServoParams servo(config.servo());
-  int16_t pin = (int16_t)servo.pin();
+  int16_t pin = static_cast<int16_t>(servo.pin());
   int16_t channel = (int16_t)pin % 4;
```

Comment:
You don't need a cast here. This is a noop because pin is already of type int16_t. Even if you mean static_cast of "pin % 4", pin is already of type int16_t, there is no way you can overflow it with % 4.

I've seen this is a pattern. You should look at the operations being performed before adding casts.